components/metadata/structured-data.tsx

```
@@ -0,0 +1,46 @@
+}
+
+/**
+ * Component for rendering JSON-LD structured data
+ * Keeps structured data logic separate from page components
+ */
+export function StructuredData({ data, id }: StructuredDataProps) {
+  return (
+    <Script
+      id={id}
+      type="application/ld+json"
+      dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)